Fill.py

```diff
@@ -500,6 +500,26 @@ def mark_for_locking(location: Location):
         items_counter.update(item.player for item in unplaced)
         print_data = {"items": items_counter, "locations": locations_counter}
         logging.info(f"Per-Player counts: {print_data})")
+        if unfilled:
+            for player in multiworld.player_ids:
+                locs_diff = locations_counter[player] - items_counter[player]
+                if locs_diff:
```

Comment:
Could potentially make this change, and the same change for `items_diff`, depends when and what errors you want exactly
```suggestion
                if locs_diff > 0:
```

Reply:
items_diff probably should be changed for consistency but i don't see any way it can possibly be less than 0 due to the other if statements